### DISCUSSION TITLE
How to read .pvd and .vtu files with pyvista

### DISCUSSION THREAD

**Discussion by rkenko** [pyvista/pyvista#2916] [Q&A]
### Describe the bug, what's wrong, and what you expected.

Hi, i wonder how to correctly read .pvd file with pyvista. I saw pyvista.PVDReader but i didn't found how to use it correctly. 

For the moment i think i know how to read .vtk files using this code:

```python 
# import modules
import numpy as np
import pyvista as pv
import matplotlib
import matplotlib.pyplot as plt

# set white theme of pyvista and scalar_bar_arguments
pv.set_plot_theme('document')  
sargs = dict(title="DATA_VALUES",title_font_size=18,label_font_size=18,italic=False,fmt="%.1f",font_family="times",vertical=True)

# plot all volume: always read before plot
result = pv.read('data.pvd')
result.plot(scalars="values",volume=True,shade=True,show_grid=True,show_axes=True,cmap="Greys_r",scalar_bar_args=sargs,show_scalar_bar=True)#,clim=[0,1])
 ```

I have the path of my .pvd and .vtu files and pv.read is not working... Please could you give me a little snippet in order to read my .pvd and .vtu files ?

Thank you in advance

### Steps to reproduce the bug.

```python 
#there is no bug 
 ```

### System Information

```shell
#there is no bug
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by MatthewFlamm** [comment]
I _think_ on the next version (0.35.0) of pyvista you will be able to use `pv.read` for pvd files.  This will read the pvd using the default reader settings, first time point, etc.

```py
import pyvista as pv
mesh = pv.read('path/to/myfile.pvd') # or myfile.vtu
```

For now, you can do this, which I recommend for pvd files, as it gives you more control.  For example, choosing time points.

```py
import pyvista as pv
reader = pv.get_reader('path/to/myfile.pvd')
# set time point to read, if needed
reader.set_active_time_value(reader.time_values[-1])
mesh = reader.read()
```

**Reply by rkenko** [reply]
Thank you very much @MatthewFlamm , it is working. However i didn't manage to display the field inside my mesh. Actually i have a field inside my mesh, just like the first example here with the bone: https://docs.pyvista.org/examples/02-plot/volume.html

I tried two codes in order to display this field:

1. This version didn't plot the field inside 
```python 
import pyvista as pv
reader = pv.get_reader('charpy_full.pvd')
# set time point to read, if needed
reader.set_active_time_value(reader.time_values[-1])
mesh = reader.read()
#plot content
mesh.plot(cmap="bone")
```

2. This versions didn't work
```python 
import pyvista as pv
reader = pv.get_reader('charpy_full.pvd')
# set time point to read, if needed
reader.set_active_time_value(reader.time_values[-1])
mesh = reader.read()
#plot content
mesh.plot(scalars="values",volume=True,shade=True,show_grid=True,show_axes=True,cmap="bone",)
```
The files i want to display are attached
[files.zip](https://github.com/pyvista/pyvista/files/9038036/files.zip)

Please @MatthewFlamm , do you know a way to display my field and the surrounding mesh using pyvista ?
